CoreBluetoothMock/CBMPeripheralSpec.swift

```diff
@@ -468,11 +494,13 @@ public class CBMPeripheralSpec {
                               services: [CBMServiceMock],
                               delegate: CBMPeripheralSpecDelegate?,
                               connectionInterval: TimeInterval = 0.045,
+                              supervisionTimeout: Double = 4.0,
```

Comment:
Could you change the type to `TimeInterval`?

Reply:
Good catch!